contents/docs/feature-flags/common-questions.mdx

```diff
@@ -43,6 +43,39 @@ import FAQFalseOrNoneEvents from "./snippets/faq-false-or-none-events.mdx"
 
 Evaluating feature flags requires making a request to PostHog. However, you can [evaluate feature flags locally](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation) instead – without having to make a request to PostHog.
 
+
+## How can I estimate the number of feature flag requests I'll be charged for?
+
+### Frontend SDKs
+
+We make a request to fetch feature flags when:
+
+1. The PostHog SDK is initialized
+2. A user is [identified](/docs/data/identify) 
+3. A user's [properties](/docs/product-analytics/user-properties) are updated.
+4. You ask us to reload feature flags (`posthog.reloadFeatureFlags()`)
+
+For the [JavaScript web SDK](/docs/libraries/js), you can estimate how many feature flag requests you will make by doing the following:
+
+1. Check the networks tab in Chrome inspector tools for the number of `/decide` requests.
+2. Find out your number of monthly page views
+3. Multiply the number your number of `/decide` requests by your monthly page views
+
+For example, if on refresh, you see 2 `/decide` requests per pageview and you have ~150,000 pageviews, your monthly feature flag requests should be around ~300,000.
+
+### Backend SDKS
+
+
+#### Without local evaluation 
+
+If you're not using [local evaluation](/docs/feature-flags/local-evaluation), a request to get feature flags is made every time you call `posthog.get_feature_flag()` on your backend. Your estimated usage will be however many times you are calling this code.
+
+#### With local evaluation
+
+If you're using [local evaluation](https://posthog.com/docs/feature-flags/local-evaluation), each local evaluation request charges 10 credits and by default are made every 30 seconds. Thus assuming your server is constantly running and making 2 requests per minute, your will be charged `10 * 2 * 60 * 24 * 30 = 864,000 credits` each month.
```

Comment:
```suggestion
If you're using [local evaluation](/docs/feature-flags/local-evaluation), each local evaluation request charges 10 credits and by default are made every 30 seconds. Assuming your server is constantly running and making 2 requests per minute, you will be charged `10 * 2 * 60 * 24 * 30 = 864,000 credits` each month.
```

Reply:
This is still such an intimidating amount to me 😅, but I guess we haven't had negative feedback on it.

Reply:
let me point you to how to reduce costs faq right below 🙈 . But yeah if we hear negative stuff I might just 10x the poll interval, which reduces the credits to 1/10th